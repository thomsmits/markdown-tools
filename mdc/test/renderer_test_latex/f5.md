# Chapter
## Slide

Die zentrale Frage ist "Wie können Menschen _Erkenntnisse_ über die Welt gewinnen?"
