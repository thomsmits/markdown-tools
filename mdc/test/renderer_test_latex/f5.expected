# Chapter
## Slide

Die zentrale Frage ist \enquote{Wie können Menschen \strong{Erkenntnisse} über die Welt gewinnen?}\vspace{0.1mm}